### PR TITLE
Qemu v8 32bits

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

File: qemu_v8.mk [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
 # S_USER (Secure User - TA) are compiled
 ################################################################################
-override COMPILE_NS_USER   := 64
+COMPILE_NS_USER ?= 64
 override COMPILE_NS_KERNEL := 64
-override COMPILE_S_USER    := 64
-override COMPILE_S_KERNEL  := 64
+COMPILE_S_USER ?= 64
+COMPILE_S_KERNEL  := 64
```
Comment (jenswi-linaro) [inline qemu_v8.mk:9]: `?=` ?

Comment (jforissier) [reply qemu_v8.mk:9]: Yes

**Comment by jforissier** [comment]
Update

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**